src/modules/design/bpmn-io/bpmn-io.scss

```
@@ -224,6 +224,24 @@
   border-radius: 2px;
 }
 
+.btn-default-red {
+  background-color: rgba(204, 51, 00, 0.95);;
```

That was my fault, I'm sorry.

```suggestion
  background-color: rgba(204, 51, 00, 0.95);
```

---

I noticed. No harm done. 😉